### PR TITLE
Add "Support Ukraine" badge in README

### PR DESCRIPTION
**Pull request by vasylnahuliak**
## Summary

This Pull Request is aimed at helping Ukrainian Army get necessary funding in time of open-scale war with Russia by providing link to official international account created by National Bank of Ukraine.

## Changelog:

[General] [Added] - Add badge with link to National Bank of Ukraine's website

## Test Plan

Badge image: 
image

### CODE REVIEW

**Comment by ZhenXiongQian** [comment]
You want to do the same thing as NATO？
Please don't get involved in politics, it's a shame like FIFA.

**Comment by j-waiting-for** [comment]
You want to do the same thing as NATO？
Please don't get involved in politics.
The open source community can't be like FIFA, that's disgusting.

**Comment by renwuqiang** [comment]
机器人自动发的，无需理会

**Comment by ErnestDong** [comment]
I've reported spam.

File: README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 The name `996.ICU` refers to **"Work by '996', sick in ICU"**, an ironic saying among Chinese developers, which means that by following the "996" work schedule, you are risking yourself getting into the ICU (Intensive Care Unit).
 
 [![Badge](https://img.shields.io/badge/link-996.icu-%23FF4D5B.svg?style=flat-square)](https://996.icu/#/en_US)
+[![Support Ukraine](https://img.shields.io/badge/Support-Ukraine-blue?style=flat&logo=adguard)](https://bank.gov.ua/en/news/all/natsionalniy-bank-vidkriv-spetsrahunok-dlya-zboru-koshtiv-na-potrebi-armiyi)
```
Comment (LitoMore) [inline README.md]: As the maintainer of Simple Icons. I must tell you that you're using [AdGuard](https://simpleicons.org/?q=adgurad)'s trademark as a normal shield icon. Which is not recommended.

Comment (vasylnahuliak) [reply README.md]: Removed logo.
Also changed the link to Nonprofits organizations.

Thanks! ☮️

**Comment by ErnestDong** [comment]
Please DO NOT impose your POLITICAL opinion on others, OK? It's disgusting to add so many spam PRs to many open source projects which is not the place you express your selfish political opinions

**Comment by bigbigDreamer** [comment]
This is the world of technologists. It has nothing to do with politics. Do what you don't want, and don't do it to others. What's the point of you putting forward PRs in all popular open source repositories? This should not be the foundation of politics, can you please go farther?

File: README.md (hunk 1 above)
Comment (LitoMore) [reply README.md]: As an open-sourcerer, I'd rather do something else meaningful than stick "ads" all over the open-source projects.

**Comment by nirui** [comment]
While I do agree that putting the badge here might be inappropriate and will probably not generate much support for Ukrainians, I also wouldn't claim such attempt as:

> stick "ads" all over the open-source projects.

It took me only 2 minutes of investigation to figure out that the original requester is likely related to a company called [eKreative], which is located in Cherkasy, Ukraine. Based on that, as I understand it, this PR could be created in some sort of desperation.

[eKreative]: https://github.com/ekreative

Attitude such as

> I'd rather do something else *meaningful* than stick "ads" ...
 
> DO NOT *impose* your *POLITICAL opinion*...

are unnecessary in this context, and to me incredibly odd for someone to obtain.

**Pull request closed without merging**